src/syntax/syntax.rkt

```diff
@@ -219,8 +219,8 @@
   (hash-set! fields 'otype otype)
   (table-set! operator-impls name fields)
   (hash-update! parametric-operators operator
-                (λ (h) (hash-set h itypes (cons name otype)))
-                (hash))
+                (curry cons (list* name otype (operator-info name 'itype)))
```

Comment:
What's the change here for?

Reply:
Related to `parametric-operators` signature change